docs/lambda_versioning.md

```diff
@@ -0,0 +1,71 @@
+# Lambda Versioning
+
+Cumulus makes use of AWS's Lambda/Alias version objects to tag and retain references to recent copies of deployed workflow lambdas.
+
+Each Cumulus deployed lambda and configured s3Source lambda in lambdas.yml will have an alias/version resource created for them.
```

Comment:
This sentence is a bit confusing. Maybe 'All Cumulus deployed lambdas in `lambdas.yml` will have an alias/version resource created. Lambdas with source coming from S3 must be expressly configured to take advantage of versioning. See <link to configuration> below.'

Reply:
👍  I like that wording.    I was avoiding that wording a bit as we *are* including a couple of internal lambdas as well, but that's not something we need to be that careful about in this doc.

Reply:
9193a5c1